src/core_plugins/tag_cloud/public/cloud_directive.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import uiModules from 'ui/modules';
+import TagCloud from 'plugins/tagcloud/tag_cloud';
+
+const module = uiModules.get('kibana/tagcloud', ['kibana']);
+
+module.directive('kbnTagCloud', function () {
+
+  return {
+    restrict: 'E',
+    scope: {
+      data: '=',
+      options: '='
+    },
+    template: '<svg class="parent"></svg>',
```

Comment:
Even though it's tiny, let's move this to a separate html file. This way, if someone wants to find all instances of the `parent` class in html files, or see where we're using `svg`, they can safely assume they only need to search `*.html`.